integ-test/src/test/java/org/opensearch/sql/ppl/StandaloneIT.java

```diff
@@ -153,6 +153,7 @@ private Settings defaultSettings() {
           new ImmutableMap.Builder<Key, Object>()
               .put(Key.QUERY_SIZE_LIMIT, 200)
               .put(Key.SQL_PAGINATION_API_SEARCH_AFTER, true)
+              .put(Key.FIELD_TYPE_TOLERANCE, false)
```

Comment:
I can see some IT set it to true and some set it to false? Maybe most IT should stick with default setting and add dedicate IT to test the opposite?

Reply:
Changed this to `true` which is the default. There is another IT `ObjectFieldSelectIT.testSelectObjectFieldOfArrayValuesItselfNoFieldTypeTolerance` that tests with field type tolerance set to `false`.